template.php

```diff
@@ -165,3 +165,22 @@ function _campaignion_foundation_exclude_block_from_contextual_links($module) {
   ];
   return in_array($module, $blacklist);
 }
+
+/**
+ * Adds data-select-two attribute to select elements.
+ *
+ * This lets the Foundation SelectTwo plugin discover the select2 elements.
```

Comment:
This isn’t connected to select2, right? If so the comment is misleading.

Reply:
it is. the Foundation plugin is named  "SelectTwo" deliberately because a naming clash:
- Foundation plugins can be autodiscovered when a DOM element has the attr `data-plugin-name`
- If the Foundation plugin (which wraps select2) would also be named "Select2" the `data-select2` attr would lead to double initialisation (as the real select2 plugin owns this attr as well)
- solution was to rename the "consumer" (the Foundation plugin)

Reply:
Ok, then the question is why we apply this to *all* selects. Regardless of whether they use select2 or not.

Reply:
With the foundation themes, we want *all* selects to use select2 so we can style them nicely (replacing the deprecated formstone selecter).

Reply:
Ok then

> This lets the Foundation SelectTwo plugin discover the select2 elements.

should rather be

> This lets the Foundation SelectTwo plugin discover the select elements.

Reply:
fine with me